js/apps/admin-ui/src/components/dynamic/components.ts

```diff
@@ -16,16 +16,25 @@ import { StringComponent } from "./StringComponent";
 import { TextComponent } from "./TextComponent";
 import { UrlComponent } from "./UrlComponent";
 import { UserProfileAttributeListComponent } from "./UserProfileAttributeListComponent";
+import { IntComponent } from "./IntComponent";
+import { NumberComponent } from "./NumberComponent";
 
 export type ComponentProps = Omit<ConfigPropertyRepresentation, "type"> & {
   isDisabled?: boolean;
   isNew?: boolean;
   stringify?: boolean;
 };
 
+export type NumberComponentProps = ComponentProps & {
+  min?: number;
+  max?: number;
+};
+
 type ComponentType =
   | "String"
   | "Text"
+  | "int"
```

Comment:
Let's keep the naming consistent and start with a capital letter and use the full name, not abbreviation.
```suggestion
  | "Integer"
```

Reply:
I originally choose `int` because it `boolean` was also written in lowercase in [ProviderConfigProperty.java](https://github.com/keycloak/keycloak/blob/main/server-spi/src/main/java/org/keycloak/provider/ProviderConfigProperty.java#L30).

I adjusted it to be "Integer" in the frontend and backend.